app/helpers/external_users/claim_types_helper.rb

```diff
@@ -0,0 +1,15 @@
+module ExternalUsers
+  module ClaimTypesHelper
+    def bill_type_prompt_heading_for(user)
```

Comment:
🗒 This helper is not very pretty, a lot of elsifs is a sign, but based on the very specific requirement of displaying a given heading for each role, this is the best I came up with for now.